src/components/overlays/quiz-overlay.js

```diff
@@ -244,16 +243,16 @@ class QuizOverlay extends BindingHelpersMixin(IocRequesterMixin(LocalizationMixi
     this._disableSubmitButton = selected.length === 0
   }
 
-  _setCurrentQuestion() {
-    if(this._currentQuestions.length > 0) {
+  _getCurrentQuestion(currentQuestions) {
+    if(currentQuestions.length > 0) {
```

Comment:
When observing array mutations as explained above, the passed argument is not the actual array, but a change record of the mutations. Therefore, you need to access the array directly with `this._currentQuestions`.

Reply:
check